ui/app/src/state/reducers/content.ts

```diff
@@ -163,39 +149,20 @@ const reducer = createReducer<ContentState>(initialState, (builder) => {
 				error: error.payload.response
 			}
 		}))
-		.addCase(fetchDetailedItem, updateItemsBeingFetchedByPath)
-		.addCase(reloadDetailedItem, updateItemsBeingFetchedByPath)
-		.addCase(completeDetailedItem, updateItemsBeingFetchedByPath)
-		.addCase(fetchSandboxItem, updateItemsBeingFetchedByPath)
-		.addCase(fetchSandboxItems, updateItemsBeingFetchedByPaths)
-		.addCase(fetchSandboxItemsComplete, (state, { payload: { items } }) => {
-			items.forEach((item) => {
-				const path = item.path;
-				state.itemsByPath[path] = parseSandBoxItemToDetailedItem(item, state.itemsByPath[item.path]);
-				delete state.itemsBeingFetchedByPath[path];
-			});
-		})
-		.addCase(fetchDetailedItemComplete, (state, { payload }) => ({
-			...state,
-			itemsByPath: {
-				...state.itemsByPath,
-				[payload.path]: payload
-			},
-			itemsBeingFetchedByPath: {
-				...reversePluckProps(state.itemsBeingFetchedByPath, payload.path)
-			}
-		}))
-		.addCase(fetchDetailedItems, updateItemsBeingFetchedByPaths)
-		.addCase(fetchDetailedItemsComplete, (state, { payload: { items } }) => {
+		.addCase(fetchContentItem, updateItemsBeingFetchedByPath)
+		.addCase(reloadContentItem, updateItemsBeingFetchedByPath)
+		.addCase(completeContentItem, updateItemsBeingFetchedByPath)
+		.addCase(fetchContentItems, updateItemsBeingFetchedByPaths)
+		.addCase(fetchContentItemsComplete, (state, { payload: { items } }) => {
 			items.forEach((item) => {
 				const path = item.path;
 				state.itemsByPath[path] = item;
 				delete state.itemsBeingFetchedByPath[path];
 			});
 		})
-		.addCase(fetchSandboxItemComplete, (state, { payload: { item } }) => {
+		.addCase(fetchContentItemComplete, (state, { payload: item }) => {
```

Comment:
Why did the payload type change? Avoid changing payload signatures, should only update the Item type

Reply:
There was this change that cause the payload type change. We used to have 4 services/actions for items retrieval:
- fetchSandboxItem: Used the bulk sandboxItems API, retrieving the first item. This is now removed.
- fetchSandboxItems: Used the bulk sandboxItems API. This is now `fetchContentItems`.
- fetchDetailedItem: Used the single item detailedItem API. This is now `fetchContentItem`.
- fetchDetailedItems: Called `x` time the detailedItem API. This is now removed.

The detail is that since `fetchSandboxItem` service and actions were removed in order to use `fetchDetailedItem` (now fetchContentItem). I kept the payload that `fetchDetailedItem` originally had. 
So basically I had to choose between the payloads since `fetchSandboxItem` and `fetchDetailedItem` were merged.